charts/kong-authorizer/values.yaml

```diff
@@ -0,0 +1,135 @@
+## kong-authorizer image parameters
+##
+image:
+  ## Image repository
+  ##
+  repository: acp.artifactory.cloudentity.com/kong-authorizer
```

Comment:
Please use `docker.cloudentity.io` instead of the abandoned `acp.artifactory.cloudentity.com`.

Reply:
Fixed.